server/modules/routing/avrorouter/avro/maxavro.cc

```diff
@@ -210,7 +210,7 @@ char* maxavro_read_string_from_file(MAXAVRO_FILE* file, size_t* size)
 }
 
 /**
- * @bref Skip an Avro string
+ * @brief Skip an Avro string
```

Comment:
There were some places where `bref` was a code token. I left that away. I'm not sure what's intended, so I left them alone.